clang/test/Driver/sycl-bc-device-libraries.cpp

```diff
@@ -0,0 +1,61 @@
+/// Test that SYCL bitcode device libraries are properly separated for NVIDIA and AMD targets.
+
+/// Check devicelib and libspirv are linked for nvptx.
+// RUN: %clang -### -fsycl --offload-new-driver \
```

Comment:
Are these test cases supported in `clang_cl` mode as well?
If yes, please update the test to add `clang_cl` test cases as well.